common/src/main/java/com/wynntils/features/user/overlays/InfoBoxesFeature.java

```diff
@@ -0,0 +1,164 @@
+                    HorizontalAlignment.Left,
+                    VerticalAlignment.Bottom);
+        }
+
+        private TextRenderTask getRenderTask() {
+            // For every & symbol, check if the next symbol is a color code and if so, replace it with §
+            // But don't do it if a \ precedes the &
+            String validColors = "0123456789abcdef";
+            StringBuilder sb = new StringBuilder(content);
+            for (int i = 0; i < sb.length(); i++) {
+                if (sb.charAt(i) == '&') { // char == &
+                    if (i + 1 < sb.length()
+                            && validColors.contains(String.valueOf(sb.charAt(i + 1)))) { // char after is valid color
+                        if (i - 1 < 0 || sb.charAt(i - 1) != '\\') { // & is first char || char before is not \
+                            sb.setCharAt(i, '§');
+                        } else if (sb.charAt(i - 1) == '\\') { // & is preceded by \, just remove the \
+                            sb.deleteCharAt(i - 1);
+                        }
+                    }
+                }
+            }
+            return new TextRenderTask(
+                    sb.toString(),
+                    TextRenderSetting.getWithHorizontalAlignment(
+                                    this.getWidth(),
+                                    CustomColor.fromChatFormatting(ChatFormatting.WHITE),
+                                    this.getRenderHorizontalAlignment())
+                            .withTextShadow(textShadow));
+        }
+
+        @Override
+        protected void onConfigUpdate(ConfigHolder configHolder) {
+            toRender = getRenderTask();
+        }
+
+        @Override
+        public void render(PoseStack poseStack, float partialTicks, Window window) {
+            FontRenderer.getInstance()
+                    .renderTextWithAlignment(
+                            poseStack,
+                            this.getRenderX(),
+                            this.getRenderY(),
+                            toRender,
+                            this.getWidth(),
+                            this.getHeight(),
+                            this.getRenderHorizontalAlignment(),
+                            this.getRenderVerticalAlignment());
+        }
+
+        @Override
+        public void renderPreview(PoseStack poseStack, float partialTicks, Window window) {
+            FontRenderer.getInstance()
+                    .renderTextWithAlignment(
+                            poseStack,
+                            this.getRenderX(),
+                            this.getRenderY(),
+                            toRender,
+                            this.getWidth(),
+                            this.getHeight(),
+                            this.getRenderHorizontalAlignment(),
+                            this.getRenderVerticalAlignment());
+        }
+    }
+
+    public static class InfoBoxOverlay1 extends InfoBoxOverlay {
```

Comment:
Why do we need these? We support multiple instances of the same overlay.

Reply:
Originally did this to separate the `content` config setting for each box without having 6 extra fields in the parent class. I will try out a different way of doing this in a bit.